Firestore/tests/Snippet/QueryTest.php

```diff
@@ -36,20 +36,15 @@ class QueryTest extends SnippetTestCase
 {
     use GrpcTestTrait;
 
+    const PARENT = 'projects/example_project/databases/(default)/documents';
```

Comment:
`parent` is a reserved keyword. In case this may cause any conflicts in the future, WDYT about updating this and other files which use this naming?